README.md

```diff
@@ -1,61 +1,51 @@
+                                       +-[ Backend Job Dyno ]--------------------                                       +
+                                       |  /jobs                                                                         |
+                                       +--------------------------------------------------------------------------------+
+```
+
+#### Build & Run
 
-If you want to run the backend you have to compile it first using `npm run build`. Make sure to have all dependencies installed before building by using `npm install`. 
+To run the backend, compile it first using `npm run build`. Make sure to have all dependencies installed before building by using `npm ci`. 
 You also need to set your environment accordingly (for further details see [.env.example](.env.example)) and set up a local database server.
 
-#### Web server
+To run the development configuration of the web server handling the API requests run `npm run web:dev`.
+The development version of the jobs can be run using `npm run jobs:dev`.
 
-To run the development configuration of the web server handling the API requests run
-```
-npm run web:dev
-```
+The `/assets` folder contains development versions of various files. During Heroku builds, the folder is replaced by a secret version that is maintained in a [separate private repository](https://github.com/corona-school/coronaschool-certificate). The commit id of the version pulled is 
+stored in `.certificate-version`. 
 
 #### Docs
 
 For building the API Docs you need [apidoc](https://apidocjs.com/) in your PATH. 
 If you use the reference from the development dependencies, you will probably have to add `node_modules/.bin` manually to your PATH. 
 If you don't mind you can also install it globally using `npm install -g apidoc`.
-The build process is initiated using
-```
-npm run web:docs
-```
-and outputs the static html to `web/public/docs`. 
+The build process is initiated using `npm run web:docs` and outputs the static html to `web/public/docs`. 
 You can either open the `index.html` manually or start the web server for serving.
 
-#### Jobs
-
-The development version of the jobs can be run using
-```
-npm run jobs:dev
-```
-
 ### Contributing
 
-We're always happy and open about contributions.
-You will find further information on how to contribute in the following paragraphs.
-
-#### What can be contributed?
-
-##### Issues
-
-While we are always happy about the contribution of issues, but our own roadmap doesn't allow us too much room to tackle requests for new features.
-
-##### Code
-
-If you want to contribute code containing fixes or new features, feel free to send us a pull request AFTER reading the information below.
-
-##### Security Issues
+We're always happy and open about contributions, please  [contact us](mailto:support@lern-fair.de) in case you are interested in joining our 
```

Comment:
team@lern-fair.de is the official mail address of HR 😉

Reply:
@lsc0498 okay should I add that or should we add a contact email for Tech directly?

Reply:
No, team@ is the best for this purpose I guess. All other applications and stuff like that also go there.